ipn/ipnlocal/local.go

```diff
@@ -4491,6 +4493,11 @@ func (b *LocalBackend) setTCPPortsInterceptedFromNetmapAndPrefsLocked(prefs ipn.
 	}
 	if b.ShouldRunWebClient() {
 		handlePorts = append(handlePorts, webClientPort)
+
+		// don't listen on netmap addresses if we're in userspace mode
```

Comment:
Nit: add why not for userspace?

Reply:
Because the kernel's never involved in that case so there can't be any "local" traffic.